cpp/include/DataStorm/DataStorm.h

```diff
@@ -626,7 +624,7 @@ namespace DataStorm
     private:
         std::shared_ptr<DataStormI::TopicReader> getReader() const noexcept;
         std::shared_ptr<DataStormI::TopicWriter> getWriter() const noexcept;
-        std::shared_ptr<Ice::Communicator> getCommunicator() const noexcept;
+        Ice::CommunicatorPtr getCommunicator() const noexcept;
```

Comment:
I switch to use the alias from Ice, to be consistent with what we do in the Ice repository as the plan is to eventually merge datastorm there.